packages/nextjs-template/components/DendronLayout.tsx

```diff
@@ -133,27 +135,32 @@ export default function DendronLayout(
         className="site-layout"
         style={{
           marginTop: 64,
-          flexDirection: "row",
         }}
       >
-        <Layout
```

Comment:
Looks like there's an issue with the styling


<img width="1920" alt="Fullscreen_9_18_21__11_29_AM" src="https://user-images.githubusercontent.com/409321/133901689-968e06e4-fe1c-4e99-8594-a5fbeafb1b20.png">

Reply:
I wrote about it in the journal and weekly.

Reply:
is this your comment about the layout being buggy?

Reply:
nvm, found the cause. it was a regression from earlier this week